doc/cephadm/certmgr.rst

```diff
@@ -0,0 +1,241 @@
+.. _orchestrator-cli-cert-management:
+
+=======================
+Certificate Management
+=======================
+
+Introduction
+============
+
+Cephadm `certmgr` acts as the Root Certificate Authority (CA) for all
+self-signed certificates generated within Cephadm. For services that require
+SSL, users have the option to either bring their own certificate or allow
+Cephadm to generate a self-signed certificate. This ensures secure communication
+while offering flexibility for different deployment preferences.
+
+Certificate Management Behavior
+===============================
+
+Cephadm `certmgr` automatically detects whether a certificate is self-signed
```

Comment:
Is this an mgr module, or functionality added to the mgr itself?

Reply:
No, it's a new  cephadm command